keyboards/ergodox_ez/keymaps/rwijtvliet/keymap.c

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 Ruud Wijtvliet (@rwijtvliet)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+// good tutorial:
+// https://precondition.github.io/home-row-mods
+
+
+#undef TAPPING_TERM
+#include "users/rwijtvliet/keymap_partial.c"
+#include "ergodox.c"
```

Comment:
C files should not be included like this.

Instead, you should use `SRC += ` in the rules.mk to include c files.